src/main/webapp/scripts/map.js

```diff
@@ -13,98 +13,85 @@
 // limitations under the License.
 
 /**
- * Map declaration and initial setup
- * At the beginning, retrieves the API_KEY from an external file
+ * Function that queries salecards from servlet.
  */
-$(function () {
-  jQuery.get('../../../../API_KEY.txt', function (textString) {
-    const API_KEY = textString;
+async function querySalecard() {
+  await updateLocation();
+
+  // Object to draw the map.
+  const clientInfo = {
+    latitude: parseFloat(document.getElementById('lat').value),
+    longitude: parseFloat(document.getElementById('lng').value),
+    radius: Number(document.getElementById('distance').value),
+    markerName: 'Your Position',
```

Comment:
nit: Content like this, the "Your Position" string, could be externalized into a single file somewhere. This makes it easier to track down and update verbiage later on. There's another one further down, the "1 vendor found."/"x vendors found.", maybe others?

Reply:
We are planning to implement something like this once we finish the MVP. A good V1 feature will be including the translation API to support street vendors around the world. Definitely I'll take this to implement our future feature.